src/api/app/models/package.rb

```diff
@@ -313,13 +313,14 @@ def master_product_object
     product_object || self
   end
 
-  def check_write_access(ignoreLock = nil)
-    User.current.can_modify_package? master_product_object, ignoreLock
+  # For the sake of having a more speaking method name...
```

Comment:
Really tiny point but do we need this comment? Once this code is merged I dont think it will really make sense..